README.md

```diff
@@ -1,5 +1,7 @@
 # Fra Negroni
 
+![FraNegroni Logo](https://repository-images.githubusercontent.com/242986480/f11d7400-5e2d-11ea-8330-aafd1f611760)
```

Comment:
the URL links to the github hosted logo of this repository.
I created a 1280x640 image (as suggested by GitHub) and uploaded it as the repo logo. This way if someone shares the repo on a social media it should display this logo